android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -285,16 +302,47 @@ public void captureEnvelope(ReadableArray rawBytes, ReadableMap options, Promise
                 }
             }
         } catch (Throwable ignored) {
-            logger.severe("Error while writing envelope to outbox.");
+            logger.log(SentryLevel.ERROR, "Error while writing envelope to outbox.");
         }
         promise.resolve(true);
     }
 
+    @ReactMethod
+    public void captureScreenshot(Promise promise) {
+
+        final Activity activity = this.getReactApplicationContext().getCurrentActivity();
```

Comment:
Can you call `getCurrentActivity()` directly here?

Reply:
I can, thanks.